tests/docker-compose-storage.yml

```
@@ -0,0 +1,31 @@
+services:
+  minio:
+    image: minio/minio:RELEASE.2023-09-20T22-49-55Z
+    command: server /data --console-address ":9001"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      interval: 5s
+      timeout: 5s
+      retries: 10
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      MINIO_ROOT_USER: distribution
+      MINIO_ROOT_PASSWORD: password
+    volumes:
+      - ./miniodata/distribution:/data
```

I needed this to support rootless on an selinux system

```suggestion
      - ./miniodata/distribution:/data:Z
```

---

what is the `Z` doing here?

---

http://manpages.org/docker-run
> “To change a label in the container context, you can add either of two suffixes :z or :Z to the volume mount. These suffixes tell Docker to relabel file objects on the shared volumes. The z option tells Docker that two containers share the volume content. As a result, Docker labels the content with a shared content label. Shared volume labels allow all containers to read/write content. The Z option tells Docker to label the content with a private unshared label. Only the current container can use a private volume.”

It shouldn't have any effect is you are not in selinux enforcing mode.